contracts/os/andromeda-kernel/src/proto.rs

```diff
@@ -20,32 +20,24 @@ pub struct IbcCounterpartyHeight {
     revision_height: Option<u64>,
 }
 
-#[derive(
-    Clone,
-    PartialEq,
-    ::prost::Message,
-    serde::Serialize,
-    serde::Deserialize,
-    schemars::JsonSchema,
-    CosmwasmExt,
-)]
-#[proto_message(type_url = "/ibc.applications.transfer.v1.MsgTransfer")]
+#[cw_serde]
```

Comment:
Remove this file + references to IBC hooks

Reply:
`on_reply_ibc_transfer` relies on `MsgTransferResponse`'s sequence. With what can we safely replace it with?